src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifMergingTest.kt

```diff
@@ -0,0 +1,31 @@
+package io.github.detekt.sarif4k
+
+import org.junit.jupiter.api.Test
+import java.io.File
+import kotlin.test.assertEquals
+
+class SarifMergingTest {
+    @Test
+    fun `merge output from sarifs of the same tool`() {
+        testMerge("/output.sarif.json", "/input_1.sarif.json", "/input_2.sarif.json")
+    }
+
+    @Test
+    fun `merge output from sarifs with different tools`() {
+        testMerge(
+            "/output_different_tool.sarif.json",
+            "/input_1.sarif.json",
+            "/input_2_different_tool.sarif.json"
+        )
+    }
+
+    private fun testMerge(expectedOutputFile: String, vararg inputFiles: String) {
+        val inputs = inputFiles.map {
+            SarifSerializer.fromJson(File(SarifSerializerTest::class.java.getResource(it)!!.path).readText())
```

Comment:
File/path is assuming an implementation detail, if the file is inside a jar this would fail.
```suggestion
            SarifSerializer.fromJson(SarifSerializerTest::class.java.getResourceAsStream(it).use { it.reader().readText() })
```

What do you think about extracting this to a function to reduce duplication and simplify testMerge method:
```
fun getTestFileContents(path: String): String =
    SarifSerializerTest::class.java.getResourceAsStream(it).use { it.reader().readText() }
```

Reply:
Thanks, I've now copied and modified existing Resource utilities from detekt tests.